app/domain/groups/primary.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2012-2023, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+module Groups
+  class Primary
+    ROLE_TYPES = [
+      Group::SektionsMitglieder::Mitglied.sti_name,
+      Group::SektionsNeuanmeldungenNv::Neuanmeldung.sti_name,
+      Group::SektionsNeuanmeldungenSektion::Neuanmeldung.sti_name
+    ].freeze
+
+    GROUP_TYPES = ROLE_TYPES.collect(&:deconstantize).freeze
+
+    def initialize(person)
+      @person = person
+      @primary_group = person.primary_group
+    end
+
+    def identify
+      preferred_roles.first&.group || roles.first&.group
+    end
+
+    def identified?
```

Comment:
`identified?` bezieht sich nur auf `preferred_roles`, `identify` kann aber auch eine non-preferred role zurückgeben.
In diesem Fall gibt `identified?` `false` zurück, aber `identify` findet trotzdem eine Rolle, was IMO semantisch ein Wiederspruch ist.
Wäre sowas wie `is_preferred?` ein besserer Name?